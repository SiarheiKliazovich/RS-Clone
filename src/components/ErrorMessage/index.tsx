import { FunctionComponent } from "react";

import img from "../../assets/images/jpg/error.jpg";

const ErrorMessage: FunctionComponent = () => {
  return (
    <img
      style={{
        display: "block",
        width: "300px",
        height: "300px",
        objectFit: "contain",
        margin: "50px auto",
      }}
      src={img}
      alt="Error"
    />
  );
};

export default ErrorMessage;
