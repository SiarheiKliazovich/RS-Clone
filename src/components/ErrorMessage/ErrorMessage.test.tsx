import { render } from "@testing-library/react";
import ErrorMessage from ".";

describe("ErrorMessage", () => {
  test("renders correctly", () => {
    const container = render(<ErrorMessage />);
    expect(container).toMatchSnapshot();
  });
});
