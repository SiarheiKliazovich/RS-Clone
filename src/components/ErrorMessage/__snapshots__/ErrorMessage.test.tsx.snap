// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders correctly 1`] = `
<div>
  <img
    alt="Error"
    src="error.jpg"
    style="display: block; width: 300px; height: 300px; object-fit: contain; margin: 50px auto;"
  />
</div>
`;
